csharp/Platform.Collections/Arrays/ArrayFiller[TElement].cs

```diff
@@ -1,37 +1,92 @@
 ﻿using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-
 namespace Platform.Collections.Arrays
 {
+    /// <summary>
+    /// <para>Represents an <see cref="T:TElement[]"/> array filler.</para>
+    /// <para>Представляет  заполнитель массива <see cref="T:TElement[]"/>.</para>
+    /// </summary>
+    /// <typeparam name="TElement"><para>The elements' type.</para><para>Тип элементов массива.</para></typeparam>
     public class ArrayFiller<TElement>
     {
         protected readonly TElement[] _array;
         protected long _position;
 
+        /// <summary>
+        /// <para> Initializes a new instance of the class <see cref="ArrayFiller"/>, using the specified array as the array to fill and the offset from which to start filling.</para></para>
```

Comment:
No need for space before `Initializes` word.

Reply:
`the <see cref="ArrayFiller"/> class` instead of `the class <see cref="ArrayFiller"/>`